templates/glue-spark-job.yaml

```diff
@@ -111,11 +111,15 @@ Resources:
         Name: !Ref JobCommand
         ScriptLocation: !Ref S3ScriptLocation
       DefaultArguments:
-        --TempDir: !Sub s3://${S3OutputBucketName}/tmp
+        --TempDir: !Sub s3://${S3BucketName}/tmp
         --enable-continuous-cloudwatch-log: !Ref ContinuousLog
+        --enable-metrics: true
+        --enable-spark-ui: true
+        --spark-event-logs-path: !Sub s3://${S3BucketName}/spark-logs/${AWS::StackName}/
         --job-bookmark-option: !Ref BookmarkOption
         --job-language: !Ref JobLanguage
         --table: !Ref GlueTableName
+        # --conf spark.sql.adaptive.enabled
```

Comment:
This is something I am starting to tweak to control the number of partitions Spark is using, which determines the number of files output, unless you coalesce.